gorush/feedback.go

```diff
@@ -0,0 +1,34 @@
+package gorush
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+)
+
+// DispatchFeedback sends a feedback to the configured gateway.
+func DispatchFeedback(log LogPushEntry) {
+
+	if PushConf.Core.FeedbackURL == "" {
+		return
+	}
+
+	payload, err := json.Marshal(log)
+
+	if err != nil {
+		LogError.Error(err)
+		return
+	}
+
+	req, _ := http.NewRequest("POST", PushConf.Core.FeedbackURL, bytes.NewBuffer(payload))
+	req.Header.Set("Content-Type", "application/json; charset=utf-8")
+
+	HTTPClient := &http.Client{}
+	resp, err := HTTPClient.Do(req)
```

Comment:
check resp state

```go
	if resp != nil {
		defer resp.Body.Close()
	}
```

Reply:
thx for the feedback. Done in [56be91e](https://github.com/appleboy/gorush/pull/414/commits/56be91e3e94448fcdfa211d1e968a3ec716b72b4)